requirements.sh

```diff
@@ -17,47 +17,44 @@
 ##--------------------------------------------------------------------
 
 ##
-## Author: Ashish Jabble, Massimiliano Pinto
+## Author: Ashish Jabble, Massimiliano Pinto, Vaibhav Singhal
 ##
 
 
 set -e
 
 foglamp_location=`pwd`
-is_rhel=`(lsb_release -ds 2>/dev/null || cat /etc/*release 2>/dev/null | head -n1 || uname -om) | egrep -o '(Red Hat|CentOS)' || echo ""`
-if [ "${is_rhel}" != "" ]; then
-	echo "Platform is ${is_rhel}"
-	centos=`echo $is_rhel | egrep 'CentOS' || echo ""`
-	set +e
-	sudo yum check-update
-	sudo yum update -y
-	set -e
-	if [ "${centos}" = "" ]; then
-		# RHEL 7 specific
-		sudo yum-config-manager --enable 'Red Hat Enterprise Linux Server 7 RHSCL (RPMs)'
+os_name=`(grep -o '^NAME=.*' /etc/os-release | cut -f2 -d\" | sed 's/"//g')`
+os_version=`(grep -o '^VERSION_ID=.*' /etc/os-release | cut -f2 -d\" | sed 's/"//g')`
+echo "Platform is ${os_name}, Version: ${os_version}"
+
+if [[ ( $os_name == *"Red Hat"* || $os_name == *"CentOS"* ) &&  $os_version == *"7"* ]]; then
+	if [[ $os_name == *"Red Hat"* ]]; then
+		yum-config-manager --enable 'Red Hat Enterprise Linux Server 7 RHSCL (RPMs)'
+		yum install -y @development
 	else
-		# CentOS 7 specific
-		sudo yum install -y centos-release-scl-rh
+		yum groupinstall "Development tools" -y
+		yum install -y centos-release-scl 
 	fi
-	sudo yum install -y @development
-	sudo yum install -y boost-devel
-	sudo yum install -y glib2-devel
-	sudo yum install -y rh-python36
-	sudo yum install -y rsyslog
-	sudo yum install -y openssl-devel
-	sudo yum install -y postgresql-devel
-	sudo yum install -y wget
-	sudo yum install -y zlib-devel
-	sudo yum install -y git
-	sudo yum install -y cmake
-	sudo yum install -y libuuid-devel
-	sudo yum install -y dbus-devel
+	yum install -y boost-devel
+	yum install -y glib2-devel
+	yum install -y rh-python36
+	yum install -y rsyslog
+	yum install -y openssl-devel
+	yum install -y postgresql-devel
+	yum install -y wget
+	yum install -y zlib-devel
+	yum install -y git
+	yum install -y cmake
+	yum install -y libuuid-devel
+	yum install -y dbus-devel
+	echo "source scl_source enable rh-python36" >> /home/${SUDO_USER}/.bashrc
```

Comment:
The PR looks good.

Just asking wether the permanent change in  /home/${SUDO_USER}/.bashrc is something the user could complain about

Reply:
If we dont do this, every time user closes his shell terminal and opens up a new, the user needs to run `scl enable rh-python36 bash`